collect_app/src/main/java/org/odk/collect/android/activities/FormHierarchyActivity.java

```diff
@@ -596,20 +602,24 @@ private void refreshView(boolean isGoingUp) {
                         break;
                     }
                     case FormEntryController.EVENT_REPEAT: {
-                        if (!formController.isGroupRelevant()) {
+                        FormEntryCaption fc = formController.getCaptionPrompt();
+                        boolean isFirst = fc.getMultiplicity() == 0;
```

Comment:
Definitely more readable to pull these all up to the top and name them. Thanks!

Reply:
Of course two of them are now unnecessary!